cmd/operator/main.go

```diff
@@ -51,6 +53,16 @@ func main() {
 	}
 	config.Credentials = credentials.NewTLSCredentials(certChainPath)
 
+	go func() {
+		healthzServer := health.NewServer(log)
+		healthzServer.Ready()
```

Comment:
This can make the status healthy before operator is not ready. Yes, normally, this should be ok. 
But, the safest way is to turn the status to ready when the actual server is ready inside the method Run of `operator.NewOperator(kubeAPI, config).Run(ctx)`

I recalled the incidents that I faced in the previous product; at that time, I used this way...

If possible, we should make the status healthy when everything is ready.

Reply:
The gRPC servers are blocking, and so there's no way to set `Ready` after the server is running.

Reply:
I pushed a change that brings the health check after the TLS certificates have been loaded and right before the server stars. If the server returns an error, we panic anyway and the Pod will restart.

Reply:
Thanks for the change. It is good to make the status healthy as late as possible..

Can we apply the same approach to injector?

Reply:
With injector its already as close as it can be, the only logic inside the `Run` method is starting a goroutine for the signals.